Gemfile

```diff
@@ -15,6 +15,7 @@ gem "rack", '>= 2.0.5'
 gem "rails-auth", '>= 2.1.4'
 gem "sinatra", '>= 2.0.4'
 
+gem "puppet", git: "https://github.com/donoghuc/puppet.git", branch: 'BOLT-1433'
```

Comment:
This fork contains a revert for disabling loaders for `Puppet[:tasks]`. I am trying to understand the type name vs task name conflict concern. For example the following plan seems to work just fine even though both the package task as well as the package type are both used in the same plan. 
```
plan test(TargetSpec $nodes) {
  # built in type
  $x = get_resources($nodes, Package)
  # loaded core type
  $y = get_resources($nodes, Cron)
  # task name same as built in type 
  $z = run_task(package, $nodes, action => status, name => vim)
  notice([$x.results, $y.results, $z.results])
```

Reply:
After discussing in person I think that the concern about name collisions between tasks and types is no longer an issue. I am still looking in to the root cause of https://tickets.puppetlabs.com/browse/BOLT-100